src/azure-cli/azure/cli/command_modules/network/generated/_params.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import resource_group_name_type
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('network express-route peering connection list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('circuit_name', type=str, help='The name of the circuit.')
```

Comment:
`circuit_name` should follow `circuit_name_type`:

https://github.com/Azure/azure-cli/blob/4fc8879d5345f150f25bc8fa41b3425d98599c4a/src/azure-cli/azure/cli/command_modules/network/_params.py#L95

Reply:
@qiaozha How to add completer for a argument?

Reply:
This has not been supported yet.

Reply:
OK, I will modify it manually